test/suites/fail-lower/L-004.pol

```diff
@@ -1,11 +0,0 @@
-data Bool { True, False }
```

Comment:
Should we move this to `fail-check` instead?

Reply:
I don't think it is a failure anymore, since locally bound variables can no longer shadow destructors. Previously we always looked up in the combined context for global and local definitions. But if we have to lower `e.d(...)` we know a priori that `d` can only be bound in the global context, so we don't attempt a lookup in the local context.

Reply:
Ah I see, nvm.